mono/metadata/gc-internals.h

```diff
@@ -313,6 +313,8 @@ int mono_gc_get_los_limit (void);
 
 guint64 mono_gc_get_allocated_bytes_for_current_thread (void);
 
+guint64 mono_gc_get_total_allocated_bytes (MonoBoolean precise);
```

Comment:
Need an implementation of this in boehm-gc.c and null-gc.c, too.

Could be just a trivial one